dashboard/app/helpers/users_helper.rb

```diff
@@ -118,10 +118,8 @@ def user_summary(user)
       user_data[:isTeacher] = true if user.teacher?
       user_data[:isVerifiedTeacher] = true if user.authorized_teacher?
       user_data[:linesOfCode] = user.total_lines
-    else
-      user_data[:linesOfCode] = client_state.lines
+      user_data[:linesOfCodeText] = I18n.t('nav.popup.lines', lines: user_data[:linesOfCode])
     end
-    user_data[:linesOfCodeText] = I18n.t('nav.popup.lines', lines: user_data[:linesOfCode])
```

Comment:
Why do we no longer need this in the case where user != nil?

Reply:
It had gone out of use. We only use linesOfCode and linesOfCodeText when the user's logged in.